Dockerfile

```diff
@@ -18,11 +18,10 @@ ENV LANG="en_US.UTF-8" \
     LANGUAGE="en_US.UTF-8" \
     LC_ALL="en_US.UTF-8"
 
-# install system dependencies
-RUN apt update -qq > /dev/null && apt install --yes --no-install-recommends \
-	python3 python3-dev virtualenv make lsb-release pkg-config git build-essential \
-    libssl-dev tox
-
 WORKDIR /app
 COPY . /app
+
+RUN apt update -qq > /dev/null && apt install --yes --no-install-recommends \
+	lsb-release make sudo && \
+	make system_dependencies
```

Comment:
Moved system dependencies to a single source of truth, the `Makefile`. That why you can install system dependencies on host or containers